Comments/ClientApp/src/components/Review/Review.js

```diff
@@ -137,13 +142,14 @@ export class Review extends Component<PropsType, StateType> {
 				sort: preloadedCommentsData.sort,
 				supportsDownload: preloadedConsultationData.consultationState.supportsDownload,
 				organisationName: preloadedCommentsData.organisationName || "",
-				respondingAsOrganisation: false,
+				respondingAsOrganisation: (preloadedCommentsData.isLead === true ? "yes" : false),
```

Comment:
this line of code is a bit of a red flag. is responding as organisation a boolean or a string? 
state says boolean